custom_model_runner/datarobot_drum/drum/args_parser.py

```diff
@@ -599,5 +600,15 @@ def verify_options(options):
             if not options.new_mode:
                 CMRunnerArgsRegistry._parsers[ArgumentsOptions.NEW].print_help()
                 exit(1)
+        elif options.subparser_name in [ArgumentsOptions.SERVER, ArgumentsOptions.PERF_TEST]:
+            if options.production:
+                ret_code = subprocess.run(["python", "-m", "pip", "show", "uwsgi"]).returncode
```

Comment:
You might be using a different python then the currently used python.. the "python argument should be the exact currently used python.. .. can you just try to load uwsgi module in the current context?